client/tailwind.config.js

```diff
@@ -5,6 +5,13 @@ module.exports = {
 		extend: {
 			colors: {
 				occ_color: "#b26910",
+				projectCardBg: "#2a2c2e",
+				projectCardDescription: "#f5f5f5",
+				onGoing: "#4CDC4C",
+				completed: "#3498db",
+				inActive: "#f1c40f",
+				canceled: "#ff2d55",
+				projectCardLinkHover: "#ff9617",
```

Comment:
Please migrate `main` to your branch and solve conflicts. A new set of colors for this project was added! The color set is reusable for the entire project. Please use it and adjust to your current tailwind code.

See pr #138 

```suggestion
				"gray-med": "#2a2c2e",
				"smoke-dark": "#CCCCCC",
				"on-going": "#4CDC4C",
				completed: "#3498db",
				"in-active": "#f1c40f",
				canceled: "#ff2d55",
				"orange": "#ff9617",
```